consensus/istanbul/backend/announce.go

```diff
@@ -41,15 +37,13 @@ import (
 // define the istanbul announce message
 
 type announceMessage struct {
-	Address               common.Address
-	IncompleteEnodeURL    string
-	EncryptedEndpointData [][][]byte
-	View                  *istanbul.View
-	Signature             []byte
+	EncryptedEnodeURLs [][][]byte
```

Comment:
Should this be `announceMessageContents`? Or something like that, to distinguish it from "messages"?

Reply:
I'll rename it to `announceMessageData`.  That conforms a bit more to how the `eth` subprotocol messages content data structures are named.